config/api/src/main/java/org/jboss/windup/config/condition/EvaluationStrategy.java

```diff
@@ -0,0 +1,27 @@
+package org.jboss.windup.config.condition;
+
+import org.jboss.windup.config.parameters.ParameterizedGraphCondition;
+import org.jboss.windup.graph.model.WindupVertexFrame;
+import org.ocpsoft.rewrite.param.ParameterValueStore;
+
+/**
+ * This is used by {@link ParameterizedGraphCondition} during parameter evaluation.
+ *
+ */
+public interface EvaluationStrategy
```

Comment:
Good. Thanks for refactoring this out. I was waiting to see if we used it everywhere. Seems like we are.